oclc/linked_data.py

```diff
@@ -64,25 +64,25 @@ def for_type(self, g, search):
 
     @classmethod
     def restrict_to_language(self, values, code_2):
-        if isinstance(values, basestring) or isinstance(values, dict):
+        if isinstance(values, (unicode, dict)):
             values = [values]
         for v in values:
-            if isinstance(v, basestring):
+            if isinstance(v, unicode):
```

Comment:
Any reference to `basestring` is problematic because Python 3 doesn't have the concept. In this file, we're only dealing with strings that were loaded from JSON, so I think it's safe to assume we're dealing with Unicode strings.

(Also, we don't use this code, it's really old and it's starting to rot -- not happy with this file in general.)